tests/client-dbus/tests/udev/_utils.py

```diff
@@ -300,12 +297,10 @@ def stop_service(self):
         :return: a tuple of stdout and stderr
         """
         self._service.send_signal(signal.SIGINT)
-        output = self._service.communicate()
+        self._service.wait()
         if list(processes("stratisd")) != []:
             raise RuntimeError("Failed to stop stratisd service")
 
-        return output
```

Comment:
Plz update docs for changed return type.

Reply:
Resolved!